README.md

```diff
@@ -9,9 +9,18 @@ You can install Ersilia Client directly from the source code:
 ```bash
 git clone https://github.com/ersilia-os/ersilia-client.git
 cd ersilia-client
-python setup.py install -e .
+pip install -e .
 ```
 
+Note: If you encounter any issues with the installation, you can try the following command:
+
+```bash
+pip install -e . --config-settings editable_mode=compat
```

Comment:
I don't think we should encourage this. This is a very simple package and should technically work without a legacy fallback.

Reply:
Sure.